package.json

```diff
@@ -11,15 +11,14 @@
   "main": "build/src/index.js",
   "typings": "build/src/index.d.ts",
   "scripts": {
-    "build-src": "npm-run-all compile-typescript webpack-plottable",
-    "build-test": "npm-run-all compile-typescript-test webpack-test",
+    "build": "npm-run-all build:tsc build:webpack",
+      "build:tsc": "tsc -p test/",
```

Comment:
the indentation was intentional to show sub-tasks (e.g. `build:tsc` is part of top-level `build`)